tests/test_rand.py

```diff
@@ -8,3 +13,62 @@ def test_random():
         assert 0 <= n <= 2 ** 64 - 1
         assert n not in m
         m.add(n)
+
+
+def test_fork():
```

Comment:
Right. This is checked below for `PYTHON_VERSION_INFO < (3, 7)` with the check `rngs == child_rngs` where there are collisions because we don't have a hook to reseed on. I added a comment to make this more obvious. 🙂